core/src/test/java/org/infinispan/commands/PutMapCommandStressTest.java

```diff
@@ -0,0 +1,212 @@
+//                              hasValue++;
+//                           }
+//                        }
+//                        if (hasValue != NUM_OWNERS) {
+//                           assertEquals("Key was " + key, NUM_OWNERS, hasValue);
+//                        }
+//                     }
+                     iteration++;
+                  }
+                  System.out.println(Thread.currentThread() + " finished " + iteration + " iterations!");
+                  return null;
+               } catch (Throwable e) {
+                  // Stop all the others as well
+                  complete.set(true);
+                  exchanger.exchange(e);
+                  throw e;
+               }
+            }
+         });
+      }
+      }
+
+      // TODO: need to figure out code to properly test having a node dying constantly
+      // Then spawn a thread that just constantly kills the last cache and recreates over and over again
+//      futures[futures.length - 1] = fork(new Callable<Void>() {
+//         @Override
+//         public Void call() throws Exception {
+//            TestResourceTracker.setThreadTestName("main");
+////            TestResourceTracker.backgroundTestStarted(PutMapCommandStressTest.this);
+//            try {
+//               Cache<?, ?> cacheToKill = cache(CACHE_COUNT - 1);
+//               while (!complete.get()) {
+//                  Thread.sleep(1000);
+//                  if (cacheManagers.remove(cacheToKill.getCacheManager())) {
+//                     log.trace("Killing cache to force rehash");
+//                     cacheToKill.getCacheManager().stop();
+//                     List<Cache<Object, Object>> caches = caches(CACHE_NAME);
+//                     if (caches.size() > 0) {
+//                        TestingUtil.blockUntilViewsReceived(60000, false, caches);
+//                        TestingUtil.waitForRehashToComplete(caches);
+//                     }
+//                  } else {
+//                     throw new IllegalStateException("Cache Manager " + cacheToKill.getCacheManager() +
+//                                                           " wasn't found for some reason!");
+//                  }
+//
+//                  log.trace("Adding new cache again to force rehash");
+//                  // We should only create one so just make it the next cache manager to kill
+//                  cacheToKill = createClusteredCaches(1, CACHE_NAME, builderUsed).get(0);
+//                  log.trace("Added new cache again to force rehash");
+//               }
+//               return null;
+//            } catch (Exception e) {
+//               // Stop all the others as well
+//               complete.set(true);
+//               exchanger.exchange(e);
+//               throw e;
+//            }
+//         }
+//      });
+
+      try {
+         // If this returns means we had an issue
+         Throwable e = exchanger.exchange(null, 5, TimeUnit.MINUTES);
+         e.printStackTrace();
```

Comment:
I find logging to be always disabled on tests and stress tests are usually ran manually so it was one less step.  I could change it over though.